pkg/cloudprovider/gce/gce.go

```diff
@@ -394,10 +394,7 @@ func fqdnSuffix() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	hostname, err := exec.Command("hostname").Output()
-	if err != nil {
-		return "", err
-	}
+	hostname := strings.Split(string(fullHostname), ".")[0]
```

Comment:
If we decide to not require fqdn suffix, this change is not required at all. Even more, we should remove the entire logic related to fqdn suffix here.